sjb/config/common/test_cases/origin_release_install_azure_39.yml

```diff
@@ -163,6 +163,34 @@ extensions:
           -e "openshift_azure_input_image_prefix=centos7" \
           playbooks/azure/openshift-cluster/launch.yml
 
+    - type: "script"
```

Comment:
I wouldn't add these in these files. My preference is a separate job from the conformance tests but @jim-minter wants them together for now (so you need to update the respective test_branch files).

Reply:
This should be the parent for the test_branch files, so it should be included, no? Or what do you mean?

Reply:
+1 on what @kargakis is saying here